Testscripts/Linux/docker_compose.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+########################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+########################################################################
+########################################################################
+#
+# Description:
+#   This script installs Docker Compose.
+#
+# Steps:
+#
+########################################################################
+
+# Function to handle the failure
+HandleFailure()
+{
+    local err_msg=$1
+    local err_code=$2
+
+    SetTestStateFailed
+    LogErr "$err_msg (ret: $err_code)"
+    exit 0
+}
+
+
+function compose_and_wordpress() {
+    mkdir wordpress
```

Comment:
there is no pass/fail judgment in this function.

Reply:
Added pass/fail check for docker compose test